pkg/resources/common.go

```diff
@@ -58,99 +54,24 @@ func suppressQuoting(_, oldValue, newValue string, _ *schema.ResourceData) bool
-		Description: "Specifies the token endpoint used by the client to obtain an access token by presenting its authorization grant or refresh token. The token endpoint is used with every authorization grant except for the implicit grant type (since an access token is issued directly).",
-	},
-	"oauth_client_auth_method": {
-		Type:             schema.TypeString,
-		Optional:         true,
-		ValidateDiagFunc: sdkValidation(sdk.ToApiAuthenticationSecurityIntegrationOauthClientAuthMethodOption),
-		Default:          "unknown",
-		Description:      fmt.Sprintf("Specifies that POST is used as the authentication method to the external service. Valid options are: %v", sdk.AsStringList(sdk.AllApiAuthenticationSecurityIntegrationOauthClientAuthMethodOption)),
-	},
-	"oauth_client_id": {
-		Type:        schema.TypeString,
-		Required:    true,
-		Description: "Specifies the client ID for the OAuth application in the external service.",
-	},
-	"oauth_client_secret": {
-		Type:        schema.TypeString,
-		Required:    true,
-		Description: "Specifies the client secret for the OAuth application in the ServiceNow instance from the previous step. The connector uses this to request an access token from the ServiceNow instance.",
-	},
-	"oauth_access_token_validity": {
-		Type:             schema.TypeInt,
-		Optional:         true,
-		ValidateFunc:     validation.IntAtLeast(0),
-		Default:          -1,
-		Description:      "Specifies the default lifetime of the OAuth access token (in seconds) issued by an OAuth server.",
-		DiffSuppressFunc: IgnoreChangeToCurrentSnowflakeValueInShow("oauth_access_token_validity"),
-	},
-	"oauth_refresh_token_validity": {
-		Type:             schema.TypeInt,
-		Optional:         true,
-		ValidateFunc:     validation.IntAtLeast(1),
-		Default:          -1,
-		Description:      "Specifies the default lifetime of the OAuth access token (in seconds) issued by an OAuth server.",
-		DiffSuppressFunc: IgnoreChangeToCurrentSnowflakeValueInShow("oauth_refresh_token_validity"),
-	},
-	"comment": {
-		Type:        schema.TypeString,
-		Optional:    true,
-		Description: "Specifies a comment for the integration.",
-	},
-	showOutputAttributeName: {
-		Type:        schema.TypeList,
-		Computed:    true,
-		Description: "Outputs the result of `SHOW SECURITY INTEGRATIONS` for the given security integration.",
-		Elem: &schema.Resource{
-			Schema: schemas.ShowSecurityIntegrationSchema,
-		},
-	},
-	describeOutputAttributeName: {
-		Type:        schema.TypeList,
-		Computed:    true,
-		Description: "Outputs the result of `DESCRIBE SECURITY INTEGRATIONS` for the given security integration.",
-		Elem: &schema.Resource{
-			Schema: schemas.DescribeApiAuthSecurityIntegrationSchema,
-		},
-	},
-}
-
-func listValueToSlice(value string, trimBrackets bool, trimQuotes bool) []string {
-	if trimBrackets {
-		value = strings.TrimLeft(value, "[")
-		value = strings.TrimRight(value, "]")
-	}
+func listValueToSlice(value string, trimQuotes bool) []string {
```

Comment:
Hmm this seems similar to what I'm adding. I moved mine to sdk/parsers.go. Maybe this should also be moved there and eventually merged, because they're doing pretty much the same thing.